### PR TITLE
Blog: How we work in Gizra.

### PR DESCRIPTION
**Pull request by DavidHernandez**
Based on my session on DrupalCamp Spain 2018

@IshaDakota or @RachelBaram this is ready to be reviewed.

### CODE REVIEW

File: src/_posts/2018-06-04-how-we-work-in-gizra.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: "How we work in Gizra"
+tags:
+  - "Drupal-planet"
```
Comment (IshaDakota) [inline src/_posts/2018-06-04-how-we-work-in-gizra.md]: I don't think we can reasonably submit this one to drupal planet ... although I guess ut was at a Drupal camp. IN any case, it needs it's own tags.

File: src/_posts/2018-06-04-how-we-work-in-gizra.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  - "Drupal-8"
+permalink: "/content/how-we-work-in-gizra/"
+layout: post
+author: DavidHernandez
+image:  "/assets/images/posts/how-we-work/thumb.jpg"
+description: "The basic building blocks to setup a usable media management suite with Drupal 8."
```
Comment (IshaDakota) [inline src/_posts/2018-06-04-how-we-work-in-gizra.md]: Needs a unique description

File: src/_posts/2018-06-04-how-we-work-in-gizra.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
+---
+title: "How we work in Gizra"
```
Comment (IshaDakota) [inline src/_posts/2018-06-04-how-we-work-in-gizra.md]: Title Case: https://titlecase.com/

File: src/_posts/2018-06-04-how-we-work-in-gizra.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+Orit is in charge of the human resources and she is the operations director.
+
+> Always try to do things better, learn from the mistakes and share with the colleagues the lessons learnt
+
+She always tries to do things better, learn from her mistakes mistakes and share the colleagues the lessons learnt, if she cans. She tries to keep it real by being upfront and transparent.
+
+Did you see any similarities between their different approaches to their daily work? I found some:
+
+### Focus on value
```
Comment (IshaDakota) [inline src/_posts/2018-06-04-how-we-work-in-gizra.md]: Title case here and all cases below.

**Comment by DavidHernandez** [comment]
@IshaDakota I think I fixed all those comments.

**Comment by IshaDakota** [comment]
@DavidHernandez Thanks! Looks great - I made proofreading edits, fixed some image links, cropped the thumbnail, and added some headings.

gizra_s_blog

fireshot capture 513 - how we work in gizra - http___localhost_3000_content_how-we-work-in-gizra_

If you approve, let's publish tomorrow.

**Comment by IshaDakota** [comment]
merged and deployed = thanks @DavidHernandez !

**Pull request merged**